### DISCUSSION TITLE
Buildroot image

### DISCUSSION THREAD

**Discussion by maitreyakale** [polarfire-soc/polarfire-soc-documentation#356] [General]
We are using buildroot tool having buildroot version 2022.02 and Icicle kit having hardware reference design version is 2023.02 , and tried to boot using SD card but the booting is not completed , it is stuck.
Please find the attached text log files for both HSS Payload and linux boot logs.
[boot_log.txt](https://github.com/polarfire-soc/polarfire-soc-documentation/files/12358115/boot_log.txt)
[hss.txt](https://github.com/polarfire-soc/polarfire-soc-documentation/files/12358116/hss.txt)

Please provide solution for the same.

**Comment by jamiegibbons** [comment]
Hi maitreyakale,

After reviewing the boot_log, I can see that you may be using incompatible versions of your components. To use Buildroot with the 2023.02 reference design, you should also be using the kernel version 5.11 from our 2023.02 release as opposed to the version 2022.11, as seen in your log.
The best way I can suggest achieving this would be to checkout the [2023.02.01 release tag for buildroot-external](https://github.com/linux4microchip/buildroot-external-microchip/tree/linux4microchip+fpga-2023.02.1). You can do this with the following command: `git clone https://github.com/linux4microchip/buildroot-external-microchip.git -b linux4microchip+fpga-2023.02.1`. Once you have done that if you do a clean build with `make clean`, `BR2_EXTERNAL=../buildroot-external-microchip/ make <your_defconfig_name>`, `make`, your image should boot. Let me know if this doesn't work and I can make further suggestions.
